CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 * The minimum supported Rust Version (MSRV) is now 1.88.0.
 
+* The Jujutsu crates `jj_lib`/`jj_cli` are now compatible with, and compiled
+  under the 2024 Edition of Rust.
```

Comment:
We can mention this, but we could also skip it. I think this change is purely jj-developer-facing; in particular 2020-edition Rust crates can still depend on jj-cli, and 2024-edition Rust crates could depend on it before.

Reply:
Yes, I think we should skip it.

Reply:
Done